### ISSUE TITLE
git/ysoserial/src/main/java/ysoserial/secmgr/DelegateSecurityManager.java:[20,4] error: method does not override or implement a method from a supertype

### ISSUE THREAD

**Issue by Meatballs1** [frohoff/ysoserial#94]
:(

**Comment by frohoff** [comment]
Can you provide the exact JDK version you are compiling with?

**Comment by Meatballs1** [comment]
10.0.01 

My workaround was just to remove the whole file for the time being

**Comment by frohoff** [comment]
Looks like a method or two got removed in 10.x and other projects are having patch their `SecurityManager` subclasses.

https://salsa.debian.org/java-team/junit4/commit/a969d35a868608447c33ed5c887acd53fd0aa927#1d374fcbee1b996627c049336acb4a3440fc760a

**Comment by frohoff** [comment]
This should be fixed as of #103.

```
$ docker run -it -v $(pwd):/app --workdir /app -v ~/.m2/:/root/.m2 maven:3.6.1-jdk-11 mvn clean compile
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------< ysoserial:ysoserial >-------------------------
[INFO] Building ysoserial 0.0.6-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ ysoserial ---
[INFO] Deleting /app/target
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ ysoserial ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /app/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ ysoserial ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 62 source files to /app/target/classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  11.432 s
[INFO] Finished at: 2019-04-25T16:30:50Z
[INFO] ------------------------------------------------------------------------
```
